File: examples/golang-push/rideshare/scooter/scooter.go
package scooter

import (
	"context"
	"rideshare/rideshare"
	"rideshare/utility"
)

func OrderScooter(ctx context.Context, searchRadius int64) {
	rideshare.Log.Printf(ctx, "ordering scooter, with searchRadius=%d", searchRadius)
	utility.FindNearestVehicle(ctx, searchRadius, "scooter")
}

File: examples/language-sdk-instrumentation/python/rideshare/django/app/ride_share/views/scooter/scooter.py
from rest_framework.views import APIView
from rest_framework.response import Response

from ride_share.utility.utility import find_nearest_vehicle

class ScooterView(APIView):
    """
    View to order a scooter
    """
    def order_scooter(self, search_radius):
        find_nearest_vehicle(search_radius, "scooter")

    def get(self, request, format=None):
        """
        Order a scooter within a certain radius.
        """
        self.order_scooter(0.3)
        return Response(["Scooter view"])
